### PR TITLE
MINOR: add table of contents

### PR DESCRIPTION
**Pull request by enothereska**
Added a simple table of contents for the developer section.

### CODE REVIEW

**Comment by enothereska** [comment]
cc @dguy thanks

File: docs/streams/developer-guide.html [hunk 1]
```diff
@@ -249,7 +299,7 @@
 </p>
 </blockquote>
 
-<h4> <a id="enable-disable-fault-tolerance-of-state-stores-store-changelogs" href="#disable-chagelogs">Enable / Disable Fault Tolerance of State Stores (Store Changelogs)</a></h4>
+<h4> <a id="disable-chagelogs" href="#disable-chagelogs">Enable / Disable Fault Tolerance of State Stores (Store Changelogs)</a></h4>
```
Comment (dguy) [inline docs/streams/developer-guide.html]: disable-changelogs

File: docs/streams/developer-guide.html [hunk 2]
```diff
@@ -299,7 +349,7 @@
 
 </pre>
 
-<h4><a id="implementing-custom-state-stores" href="#implement-custom-store">Implementing custom State Stores</a></h4>
+<h4><a id="implementing-custom-state-stores" href="#implementing-custom-state-stores">Implementing custom State Stores</a></h4>
```
Comment (dguy) [inline docs/streams/developer-guide.html:352]: nice one!

File: docs/streams/developer-guide.html [hunk 3]
```diff
@@ -331,7 +381,7 @@
   The <code>StateRestoreListener</code> in this case is per state store instance and is used for internal purposes such as updating config settings based on the status of the restoration process.
 </p>
 
-<h4><a id="connecting-processors-and-state-stores" href="#connecting-processors-state-stores">Connecting Processors and State Stores</a></h4>
+<h4><a id="connecting-processors-and-state-stores" href="#connecting-processors-and-state-stores">Connecting Processors and State Stores</a></h4>
```
Comment (dguy) [inline docs/streams/developer-guide.html:384]: +1

File: docs/streams/developer-guide.html [hunk 4]
```diff
@@ -1956,7 +2006,7 @@
         </li>
     </ul>
 
-    <h4><a id="treams_developer-guide_interactive-queries_your_app" href="#treams_developer-guide_interactive-queries_your_app">Your application and interactive queries</a></h4>
+    <h4><a id="streams_developer-guide_interactive-queries_your_app" href="#streams_developer-guide_interactive-queries_your_app">Your application and interactive queries</a></h4>
```
Comment (dguy) [inline docs/streams/developer-guide.html:2009]: +1

File: docs/streams/developer-guide.html [hunk 5]
```diff
@@ -2646,7 +2696,7 @@
     StreamsConfig config = new StreamsConfig(settings);
     </pre>
 
-    <h4><a id="streams_client_config" href="#streams_clients_config">Producer and Consumer Configuration</a></h4>
+    <h4><a id="streams_client_config" href="#streams_client_config">Producer and Consumer Configuration</a></h4>
```
Comment (dguy) [inline docs/streams/developer-guide.html:2699]: +1

**Review by dguy — COMMENTED** [review]
Thanks @enothereska, just one change needed otherwise LGTM!

**Comment by enothereska** [comment]
Thanks @dguy

**Comment by dguy** [comment]
Thanks @enothereska! Merged to trunk

**Pull request closed without merging**